pages/registration-role.js

```diff
@@ -38,20 +39,22 @@ const RegistrationRole = (props) => (
           >
             <Radio
               name="registration_role"
```

Comment:
a global search for registration_role reveals a few tests where it's not assigned the new enum value. Whilst we might get away with it, might be confusing in 6 months. Same with other enum fields I suspect.

Reply:
Updated tests to use enums where appropriate